"0",""
"0","t_test_results <- t.test(difference_scores)"
"0",""
"0","lower_interval<- t_test_results$conf.int[1]"
"0",""
"0","upper_interval<- t_test_results$conf.int[2]"
"0",""
"0",""
"0","ggplot(x=""MeanDifference"", y=mean_difference)+"
"0","  geom_bar(stat=""identity"", width=.5, alpha=.5)+"
"0","  geom_hline(yintercept=0)+"
"0","  geom_point(aes(y=difference_scores), alpha=.25)+"
"0","  geom_errorbar(aes(ymin=lower_interval, "
"0","                                  ymax=upper_interval), width=.1)"
"1","[1m[33mError[39m in `geom_point()`:[22m
[38;5;255m[33m![38;5;255m Problem while setting up geom.
[36mâ„¹[38;5;255m Error occurred in the 3rd layer.[39m
[1mCaused by error in `compute_geom_1()`:[22m
[38;5;255m[33m![38;5;255m `geom_point()` requires the following missing
  aesthetics: [32mx[38;5;255m[39m
Backtrace:
[90m  1. [39m[1mbase[22m (local) `<fn>`(x)
[90m  2. [39mggplot2:::print.ggplot(x)
[90m  4. [39mggplot2:::ggplot_build.ggplot(x)
[90m  5. [39mggplot2:::by_layer(...)
[90m 12. [39mggplot2 (local) f(l = layers[[i]], d = data[[i]])
[90m 13. [39ml$compute_geom_1(d)
[90m 14. [39mggplot2 (local) [1m[94mcompute_geom_1([39m[22m..., self = self[1m[94m)[39m[22m
"
